modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc
+// * rosa_getting_started/quickstart.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-sts-creating-account-wide-sts-roles-and-policies_{context}"]
+= Creating the account-wide STS roles and policies
+
+Before using the {cluster-manager-first} {hybrid-console-second} to create {hcp} for {product-title} (ROSA) clusters that use the AWS Security Token Service (STS), create the required account-wide STS roles and policies, including the Operator policies.
+
+.Prerequisites
+
+* You have completed the AWS prerequisites for {hcp} for ROSA with STS.
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You have installed and configured the latest ROSA CLI (`rosa`) on your installation host.
++
+[NOTE]
+====
+To successfully install {hcp} for ROSA clusters, use the latest version of the ROSA CLI.
+====
+* You have logged in to your Red Hat account by using the `rosa` CLI.
+
+.Procedure
+
+. Check your AWS account for existing roles and policies:
++
+[source,terminal]
+----
+$ rosa list account-roles
+----
+
+. If they do not exist in your AWS account, create the required account-wide STS roles and policies. Optionally, you can specify a prefix for the account roles:
++
+[source,terminal]
+----
+$ rosa create account-roles --mode auto -y --version 4.12 [--prefix <something>]
+----
```

Comment:
```suggestion
. If they do not exist in your AWS account, create the required account-wide STS roles and policies:
+
[source,terminal]
----
$ rosa create account-roles
----
```

Just aligning with the more generic command used in other docs, https://55565--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.html#clipboard-8